docs/guides/grafana-vmgateway-openid-configuration.md

```diff
@@ -0,0 +1,162 @@
+# How to configure vmgateway for multi-tenant access using Grafana and OpenID Connect
+
+Using Grafana with vmgateway is a great way to provide multi-tenant access to your metrics.
+vmgateway provides a way to authenticate users using JWT tokens issued by an external identity provider.
+Those tokens can include information about the user and the tenant they belong to, which can be used
+to restrict access to metrics to only those that belong to the tenant.
+
+## Prerequisites
+
+* Identity service that can issue JWT tokens
+* Grafana
+* VMSingle or VMCluster
+* vmgateway
+
+## Configure identity service
+
+The identity service must be able to issue JWT tokens with the following `vm_access` claim. Here is an example claims
+object:
+
+```json
+{
+  "vm_access": {
+    "tenant_id": {
+      "account_id": 0,
+      "project_id": 0
+    }
+  }
+}
+```
+
+You can find details about all supported options in the
+vmgateway [documentation](https://docs.victoriametrics.com/vmgateway.html#access-control).
+
+### Configuration for Keycloak
```

Comment:
what is Keycloak? it wasn't mentioned before

Reply:
It's an opensource identity provider commonly used for SSO installations. Added a note and reference into guide